config-model/src/main/java/com/yahoo/searchdefinition/RankProfile.java

```diff
@@ -214,7 +217,8 @@ private RankProfile getInherited() {
                 if (search.getDeployProperties().featureFlags().enforceRankProfileInheritance()) {
                     throw new IllegalArgumentException(msg);
                 } else {
-                    log.warning(msg);
+                    deployLogger.log(Level.WARNING, msg);
```

Comment:
```suggestion
                    deployLogger.logApplicationPackage(Level.WARNING, msg);
```
To show this as a notification in console, e.g.:
![Screenshot from 2021-09-08 18-25-41](https://user-images.githubusercontent.com/3785505/132547847-87c8befe-3809-480a-a4cf-2b4ed6f13d74.png)

Reply:
Thanks, will do.